.github/workflows/release.yml

```
@@ -0,0 +1,73 @@
+name: release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false # otherwise, the token used is the GITHUB_TOKEN, instead of your personal token
+          fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
+      - name: Auto Increment Semver Action
+        uses: MCKanpolat/auto-semver-action@1.0.5
+        id: versioning
+        with:
+          releaseType: patch 
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Create Github Release
+        uses: actions/create-release@v1
+        if: steps['versioning']['outputs']['RETURN_STATUS'] == '0'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.versioning.outputs.version }}
+          release_name: ${{ steps.versioning.outputs.version }}
+          body: Version ${{ steps.versioning.outputs.version }}
+          draft: false
+          prerelease: false
+      - name: Update pulp-core pin
+        run: sed -i "s,FROM kong/pulp-core.*,FROM kong/pulp-core:${{ steps.versioning.outputs.version }},g" */Dockerfile
```

```suggestion
        run: sed -i 's,\(FROM kong/pulp-core\):?.*$,\1${{ steps.versioning.outputs.version }},g' */Dockerfile
```
I really like using single quotes for shell safety and a match reference to avoid data duplication in this sort of thing.

I don't like the comma either. But before I go down that road... Does this step actually need to happen? If the build sequence below were to build "latest" first, then pulp-core would be built and available locally when the derived containers are built. The tagged version build will all be using cached when it re-runs, so it'll be ok too.  That would allow skipping the commit back as well, keeping stuff clean.

---

so I added this step because super-linter complained that the `FROM` should be tagged and pedantically it is correct :(

---

Could we just tag as :latest and still skip the sed / recommit?

---

heh nope `:latest` also makes the linter unhappy and again "technically" best practice says to pin things and let renovate for example update.

Letting renovate update when we're the ones doing the releases kind of seemed like going around the block to get next door